native/cocos/platform/mac/View.mm

```diff
@@ -268,14 +268,45 @@ - (BOOL)acceptsFirstResponder {
 }
 
 - (void)sendMouseEvent:(int)button type:(cc::MouseEvent::Type)type event:(NSEvent *)event {
-    const NSRect contentRect = [self frame];
-    const NSPoint pos = [event locationInWindow];
-
     _mouseEvent.windowId = [self getWindowId];
     _mouseEvent.type = type;
     _mouseEvent.button = button;
-    _mouseEvent.x = pos.x;
-    _mouseEvent.y = contentRect.size.height - pos.y;
+    
+    auto *windowMgr = CC_GET_PLATFORM_INTERFACE(cc::SystemWindowManager);
+    auto *window = dynamic_cast<cc::SystemWindow*>( windowMgr->getWindowFromNSWindow([self window]));
+    const NSRect contentRect = [self frame];
+    if(!window->isPointerLock()) {
+        const NSPoint pos = [event locationInWindow];
+        _mouseEvent.x = pos.x;
+        _mouseEvent.y = contentRect.size.height - pos.y;
+    } else {
+        if(type == cc::MouseEvent::Type::MOVE) {
+            _mouseEvent.x = _mouseEvent.x + [event deltaX];
+            _mouseEvent.y = _mouseEvent.y + [event deltaY];
+            float x_min = 0, x_max = 0;
+            float y_min = 0, y_max = 0;
```

Comment:
Please use `xMin`, `xMax` instead for parameters.

Reply:
OK